### ISSUE TITLE
 Box(P          R      mAP50  mAP50-95)

### ISSUE THREAD

**Issue by WanderingSta** [ultralytics/ultralytics#8665]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### YOLOv8 Component

Train

### Bug

为什么在YOLOV8 上训练的时候， Box(P          R      mAP50  mAP50-95) 这些参数都为0？怎么修改？

Why is Box(PR mAP50 MAP5
![屏幕截图 2024-03-05 103544](https://github.com/ultralytics/ultralytics/assets/112313941/4b44ae50-cf96-428f-9595-a4d4ef0a20be)
0-95) set to 0 when training on YOLOV8? How to modify?

### Environment

yolo task=detect mode=train model=F:\myYOLOV8\ultralytics\cfg\models\v8\yolov8_att.yaml data=coco8.yaml epochs=100 batch=16 

### Are you willing to submit a PR?

- [X] Yes I'd like to help by submitting a PR!

**Comment by glenn-jocher** [comment]
@WanderingSta hello! 👋 It looks like you're encountering an issue where your training metrics are showing as zeros. This could be due to several reasons, such as improper dataset labeling, incorrect dataset path, or the model not learning from the data. Here are a couple of things you might want to check:

1. **Dataset and Labels**: Ensure your dataset is correctly labeled and the paths in your `coco8.yaml` file are correctly pointing to your dataset locations.
2. **Learning Rate**: Sometimes, if the learning rate is too high or too low, the model might not learn effectively. You could try adjusting the learning rate.
3. **Model Configuration**: Double-check the `yolov8_att.yaml` configuration to ensure it's suitable for your dataset size and complexity.

Here's a quick example of how you might adjust the learning rate in your training command:

```bash
yolo task=detect mode=train model=F:\myYOLOV8\ultralytics\cfg\models\v8\yolov8_att.yaml data=coco8.yaml epochs=100 batch=16 lr0=0.01
```

If these steps don't resolve the issue, it might be helpful to share more details about your dataset and training logs for further assistance. 😊

And since you're willing to contribute by submitting a PR, it would be fantastic if you could share any findings or improvements you make during this process. Your contributions are highly appreciated by the community!

**Comment by WanderingSta** [comment]
@glenn-jocher Thanks!!!!!!

**Comment by glenn-jocher** [comment]
@WanderingSta you're welcome! 😊 If you have any more questions or need further assistance, feel free to ask. Happy coding!
